Assets/Source/Components/Actor/ActorBehavior.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using Assets.Source.Components.Base;
```

Comment:
A good way to think of this solution, is if the player cannot function without a HUD, then the code is not correct.  I can think of tons of situations where the player would exist without the HUD, and having to do hacky things like hide the HUD sounds not ideal.